src/server/helpers/achievement.js

```diff
@@ -113,7 +118,10 @@ function awardTitle(editorId, tier) {
 						const out = {};
 						out[tier.titleName] = unlock;
 						return out;
-					});
+					})
+					.catch((err) =>
+						console.log(err)
```

Comment:
Its not something I can test in the actual application since its unlikely that an incorrect editor Id will get passed in. But it caused a massive problem in writing unit tests since I was getting an UnlockNotFound error, which was hiding the actual issue of the editorId not being found, I Will document this to make note of it, but I personally think it should stay.